cli.js

```diff
@@ -116,6 +116,11 @@ if (cli.flags.init) {
 	return;
 }
 
+if (cli.flags.watch && cli.flags.tap && !conf.tap) {
```

Comment:
My thought was that a user might try both flags trying to troubleshoot why tap wasn't working in watch mode. I thought an explicit error would be better than hoping they had read the docs